### ISSUE TITLE
Dragon Graphics Error

### ISSUE THREAD

**Issue by Euphratis88** [Alex-the-666/Ice_and_Fire#2700]
Good evening guys.

After installing the latest version for 1.15.2, I noticed a graphics error with the dragons. A black square texture appears under the wings, running the length of the wings. I can provide pictures or a video if this helps? I tried to removed texture packs but didn't work. I'll try some other changes in settings to see if it helps. Any idea what could be causing this?

Thank you for your time and effort you put into this mod!

Regards

Si

**Comment by AlexModGuy** [comment]
image please?

**Comment by Euphratis88** [comment]
![2020-09-07_22 57 19](https://user-images.githubusercontent.com/70914246/92418082-9c044700-f15d-11ea-825e-90da335ac687.png)
![2020-09-07_22 57 07](https://user-images.githubusercontent.com/70914246/92418083-9d357400-f15d-11ea-93b9-80be5da0d892.png)
![2020-09-07_22 57 10](https://user-images.githubusercontent.com/70914246/92418084-9e66a100-f15d-11ea-8f0f-0b2bc78f4eec.png)

**Comment by AlexModGuy** [comment]
Is this on every dragon? Or only a few colors? And what about without shaders?

**Comment by Euphratis88** [comment]
It's not constant though, it sort of flickers. Yes, every dragon I have seen so far, Fire, Ice and Lighting.

**Comment by Carro1001** [comment]
does this happen with any shader?

**Comment by Euphratis88** [comment]
![2020-09-07_23 01 36](https://user-images.githubusercontent.com/70914246/92418194-277dd800-f15e-11ea-8569-063b50e78902.png)
![2020-09-07_23 01 06](https://user-images.githubusercontent.com/70914246/92418196-29479b80-f15e-11ea-9bd0-249e6ef97955.png)


Hmm, I'll turn off the shaders and let you know asap.

**Comment by Euphratis88** [comment]
Yes, it looks like its the shaders.

**Comment by Euphratis88** [comment]
I have fixed it. After figuring out it was the shaders. I looked at their website and saw there was a recent update. Installed that and gave it a go. All sorted.

Original shaders were: Sildurs Vibrant Shaders v1.27 Lite

Updated to:

Sildurs Vibrant Shaders v1.28 Lite

Hopefully if anyone else has this issue they can sort it out :)

Thank you again for your time Alex. Amazing work you have done here!
